### ISSUE TITLE
Is it possible for users to choose what address format to use with Bitcoin Cash?

### ISSUE THREAD

**Issue by KainniaK** [Blockchair/Blockchair.Support#107]
Right now with Bitcoin Cash I can only see cash addr, can there not be a config button where I can choose to see the original address format? I use a lot of vanity gen addresses and now I can't see those anymore.

**Comment by kk999999999** [comment]
I know how you can do I Gef your 
Later ok

**Comment by Har01d** [comment]
Hello,

We display addresses in 3 formats on the address dashboard page, e.g. https://blockchair.com/bitcoin-cash/address/qp3wjpa3tjlj042z2wv7hahsldgwhwy0rq9sywjpyy

It’s also possible to search for old address format: https://blockchair.com/search?q=1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa

**Comment by KainniaK** [comment]
Yes but when I click through output and addresses it's all done in cash addr.

When I browse through blocks and outputs and inputs I can immediately recognize my own addresses like 1Niak6WPi1NDYquYGBc6TYVerheyEGeXM7 with a pattern in beginning and end.

Now I need to go to blocktrail for that, and your site is a way better. Electron Cash also only allows me to use blockchair.

So it would be really nice to give a user the option in what address format he wants to browse the site.

**Comment by dharma7777** [comment]
Sir me plz help.
Sir me withdraw a wrong wallet address.
So me bye mistake a Bitcoin cash wallet. Sir my address change possible
Bitcoin cash to Bitcoin .
Sir my amount blockchain show but my address wrong .
Sir my private Key not found and my transaction 60 after unconfirmed.
Sir plz my help.
Me wrong Bitcoin cash wallet addres------3DUMfv2SV4sNDEduZd6Kb9y9EB4Kayq7su
My wrong wallet  transactionID--781a44ad578dc2eea19f898f3813ab9c94bbd629e3eed891848ca4e17a07aa60

**Comment by kk999999999** [comment]
content://com.google.android.apps.docs.storage.legacy/enc%3DhJjSa9FGxGOmWUX84L5V3UhaJNR5OBaoa6qEhQyHBRz4_4j_%0A
